messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/deployment/JMSDestinationDefinitionInjectionSource.java

```diff
@@ -189,7 +190,7 @@ private void startQueue(final String queueName,
         final boolean durable = properties.containsKey(DURABLE.getName()) ? Boolean.valueOf(properties.get(DURABLE.getName())) : DURABLE.getDefaultValue().asBoolean();
         final String managementAddress =  properties.containsKey(MANAGEMENT_ADDRESS.getName()) ? properties.get(MANAGEMENT_ADDRESS.getName()) : MANAGEMENT_ADDRESS.getDefaultValue().asString();
         final String user =  properties.containsKey("management-user") ? properties.get("management-user") : null;
-        final String password =  properties.containsKey("management-password") ? properties.get("\"management-password") : null;
+        final String password =  properties.containsKey("management-password") ? properties.get("management-password") : null;
```

Comment:
This overlaps fix made in subsequent https://github.com/wildfly/wildfly/pull/14416